crates/fuel-core/src/graph-api/mod.rs

```diff
@@ -0,0 +1,21 @@
+use fuel_core_types::{
```

Comment:
no mod.rs, please keep consistent with rust 2018 modules.

Reply:
I'd prefer to keep the module style consistent for now, and maybe add a note to move this file to lib.rs in a separate commit to avoid losing the history.